content/chainguard/libraries/faq.md

```diff
@@ -12,6 +12,71 @@ weight: 003
 toc: true
 ---
 
+## What security issues can Chainguard Libraries prevent?
+
+As detailed in the [background](/chainguard/libraries/overview/#background) and
+[introduction](/chainguard/libraries/overview/#introduction) Chainguard
+Libraries are built directly from source in the Chainguard Factory and the
+resulting binaries are directly provided to you by Chainguard. Chainguard
+operates the whole supply chain for the package lifecycle as one reliable,
+secure partner. You can therefore avoid issues from the following software
+supply chain attacks surface points:
+
+* Build pipeline
+* Build system
+* Dependency injection
+* Bypass of CI/CD systems
+* Library distribution
+* Library consumption
+
+More information about the stages is available on the [Supply chain Levels for
+Software Artifacts (SLSA) website](https://slsa.dev/).
+
+Following are examples that exploited issues in the relevant supply
+chain:
+
+**Example: Ultralytics Python project**
```

Comment:
I don't think we need to include "Example", it's assumed

```suggestion
**Ultralytics Python project**
```

Reply:
Done here and for others